### PR TITLE
add python wrapper for sequence_pool

### LINKED ISSUE #6777: Need add python wrapper for sequence_pool
We should add three python wrapper for sequence_pool op.
- sequence_first_step
- sequence_last_step
- sequence_pool: whose pool_type is avg/max/sum/sqrt

### PR DESCRIPTION
**Pull request by luotao1**
fix #6777 
The generated sequence_pool website is : 
image
image
image

### CODE REVIEW

**Comment by luotao1** [comment]
The generated sequence_first_step website is :
image
image
The generated sequence_last_step website is :
image
image

File: python/paddle/v2/fluid/layers/nn.py [hunk 1]
```diff
@@ -520,9 +520,53 @@ def _get_default_param_initializer():
...
+    Examples:
+
+        .. code-block:: python
+             
+             x = fluid.layers.data(name='x', shape=[7, 1], 
+                              dtype='float32', lod_level=1)
+             avg_x = fluid.layers.sequence_pool(input=x, pool_type='average')
+             sum_x = fluid.layers.sequence_pool(input=x, pool_type='sum')
+             sqrt_x = fluid.layers.sequence_pool(input=x, pool_type='sqrt')
+             max_x = fluid.layers.sequence_pool(input=x, pool_type='max')
```
Comment (pkuyym) [inline python/paddle/v2/fluid/layers/nn.py]: Is it necessary to limit the type in (average/sum/sqrt/max)?
If so, I think it's better to add a check.

Comment (luotao1) [reply python/paddle/v2/fluid/layers/nn.py]: If I limit the type in (average/sum/sqrt/max), then sequence_first_step should not be
```
def sequence_first_step(input, **kwargs):
    return sequence_pool(input=input, pool_type="first")
```
It should be 
```
def sequence_first_step(input, **kwargs):
    helper = LayerHelper('sequence_pool', input=input, **kwargs)
    dtype = helper.input_dtype()
    pool_out = helper.create_tmp_variable(dtype)
    max_index = helper.create_tmp_variable(dtype)

    helper.append_op(
        type="sequence_pool",
        inputs={"X": input},
        outputs={"Out": pool_out,
                 "MaxIndex": max_index},
        attrs={"pooltype": "first"})

    return pool_out
```
but `max_index = helper.create_tmp_variable(dtype)` could not be removed. If we want to remove it, we should add an single sequence_max_op.cc

Comment (pkuyym) [reply python/paddle/v2/fluid/layers/nn.py]: I see, thanks.

**Pull request merged**